### ISSUE TITLE
nothing happens when I launch through extensions menu

### ISSUE THREAD

**Issue by mcardeiro** [Adobe-CEP/Samples#104]
# Submit an issue

Note: issues submitted on this repo should be related to the samples contained within. If you have a general question about CEP, ExtendScript, or related topics, please visit the [Extensions development forum](https://forums.adobe.com/community/creative_cloud/add-ons/extensions).

## Topic

This is an issue regarding:

- [ ] The samples contained within this repo.
- [ ] The documentation contained within this repo.

## Versions

- [ ] CEP version(s) used [must be `8.x+` for this repo]:
- [ ] Supported CC app(s) and version(s):

**Comment by dkstevekwak** [comment]
@mcardeiro which sample did you try to use?

**Comment by mcardeiro** [comment]
PProPanel

On Tuesday, March 19, 2019, 11:24:10 AM EDT, dkstevekwak <notifications@github.com> wrote:  
 
 
@mcardeiro which sample did you try to use?

**Comment by dkstevekwak** [comment]
@mcardeiro have you followed the instructions noted in https://github.com/Adobe-CEP/Samples/blob/master/PProPanel/ReadMe.md? especially step #2 and #3. Also, if you enable debugging by following the fourth step, you should be able to debug further

**Comment by mcardeiro** [comment]
I did everything but step 4.  Just did step 4, tried opening panel (didnt open). went to browser and typed localhost: 7777 and got a file can't be reached error.


    On Tuesday, March 19, 2019, 11:32:45 AM EDT, dkstevekwak <notifications@github.com> wrote:  
 
 
@mcardeiro have you followed the instructions noted in https://github.com/Adobe-CEP/Samples/blob/master/PProPanel/ReadMe.md? especially step #2 and #3. Also, if you enable debugging by following the fourth step, you should be able to debug further

**Comment by bbb999** [comment]
If the extension appears in the menu but cannot be opened, then the loading of debug panels (PlayerDebugMode) has not been enabled for this version of PPro.

Try enabling PlayerDebugMode for both CSXS.8 (PPro 11 and 12), and CSXS.9 (PPro 13).

**Comment by mcardeiro** [comment]
just enabled for CSXS.8, restarted premiere and tried to launch panel and...same, nothing happens
    On Tuesday, March 19, 2019, 1:57:10 PM EDT, Bruce Bullis <notifications@github.com> wrote:  
 
 
If the extension appears in the menu but cannot be opened, then the loading of debug panels (PlayerDebugMode) has not been enabled for this version of PPro.

Try enabling PlayerDebugMode for both CSXS.8 (PPro 11 and 12), and CSXS.9 (PPro 13).

**Comment by ErinFinnegan** [comment]
Different versions of Chrome can cause problems. Which one are you using?

**Comment by bbb999** [comment]
That's odd; send me the panel? [day job address = b b b at adobe dot com]

**Comment by jmatt04** [comment]
Hello, I'm having the same issue. Adobe Premiere Pro 2018 (12.0), followed the instructions as well. nothing happens when the sample is invoked via (Premiere Pro) Window > Extensions > PProPanel (SDK Sample Panel).

Also, I'm using Chrome Version 73.0.3683.86 (Official Build) (64-bit)

I'll update if I get mine working. Thanks!

**Comment by bbb999** [comment]
Behavior when PlayerDebugMode isn't set = The sample appears on the menu but does nothing when invoked. jmatt04, have you restarted the system since setting PlayerDebugMode, for CSXS.8?

**Comment by jmatt04** [comment]
I got it working! Here's what I did. I tried restarting the system, it didn't work. I then updated my Premiere to 12.1. Still it didn't work. I then tried using the 12.1 sample as I've updated the Premiere, and still it didn't work. I then deleted the registry entry for PlayerDebugMode and created a new one. basically the same. For some reason it did work lol. I think what made it work was redoing the PlayerDebugMode step. Thanks!
